rules/linux/execution_c89_c99_binary.toml

```
@@ -25,9 +25,7 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-sequence by host.id, process.pid with maxspan=1m
-[process where process.name :("c89","c99") and process.args == "-wrapper" and process.args : ("sh,-s", "bash,-s", "dash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s")]
-[process where process.parent.name : ("bash", "sh", "dash")]
+process where event.type == "start" and process.name in ("sh", "dash", "bash") and process.parent.name == "gcc" and process.parent.args == "-wrapper" and process.parent.args in ("-std=c99", "-std=c89") and process.parent.args in ("sh,-s", "bash,-s", "dash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s") 
```

This rule looks to still be unintentionally changed / muddled with another rule

```suggestion
process where event.type == "start" and process.name in ("sh", "dash", "bash") and 
  process.parent.name in ("c89","c99") and process.parent.args == "-wrapper" and 
  process.parent.args in ("sh,-s", "bash,-s", "dash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s") 
```